sortdb/metadata.csv

```diff
@@ -0,0 +1,29 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration
+sortdb.stats.total_requests,GAUGE,,task,,Total number of requests,0,sortdb
+sortdb.stats.total_seeks,GAUGE,,task,,Total number of seeks,0,sortdb
+sortdb.stats.get_requests ,GAUGE,,task,,Total number of get requests,0,sortdb

```

Comment:
There is an extra space on a few of these lines after the metric name. Not sure if the metrics metadata pipeline will play nicely, can we remove those?